modules/bmo-about-the-hostupdatepolicy-resource.adoc

```diff
@@ -0,0 +1,34 @@
+// This is included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="bmo-about-the-hostupdatepolicy-resource_{context}"]
+= About the HostUpdatePolicy resource
+
+You can use the `HostUpdatePolicy` resource to enable or disable applying live updates to the firmware settings, BMC settings, or BIOS settings of each bare-metal host. By default, the operator disables live updates to already provisioned bare-metal hosts by default.
+
+.The HostUpdatePolicy spec
+
+The `spec` section of the `HostUpdatePolicy` resource provides two settings:
+
+* `firmwareSettings`: This setting corresponds to the `HostFirmwareSettings` resource. 
+
+* `firmwareUpdates`: This setting corresponds to the `HostFirmwareComponents` resource.
```

Comment:
See the ISG for guidance on when to use definition lists: https://www.ibm.com/docs/en/ibm-style?topic=format-lists#definition-lists

Reply:
Converted to definition list.